sim/paladin/retribution/retribution_test.go

```diff
@@ -216,6 +233,21 @@ var Phase3Consumes = core.ConsumesCombo{
 		ZanzaBuff:         proto.ZanzaBuff_AtalaiMojoOfWar,
 	},
 }
+var Phase4Consumes = core.ConsumesCombo{
+	Label: "Phase 4 Consumes",
```

Comment:
Small knit, should this be phase 5?

Reply:
I prefer it as phase 4, since I'm not aware of any consumables that were added in phase5